Description: Disable dependency winapi which is used only in doc examples
Forwarded: not-needed
Last-Update: 2024-12-20
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -56,5 +56,5 @@
 
-[dev-dependencies.winapi]
-version = "0.3"
-features = ["winbase"]
+#[dev-dependencies.winapi]
+#version = "0.3"
+#features = ["winbase"]
 
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -124,3 +124,3 @@
 //!
-//! ```rust
+//! ```rust,ignore
 //! # #[cfg(any(not(windows), not(feature = "alloc")))]
@@ -168,3 +168,3 @@
 //!
-//! ```rust
+//! ```rust,ignore
 //! # #[cfg(any(not(windows), not(feature = "alloc")))]
